packages/nx/src/utils/package-manager.ts

```diff
@@ -75,7 +75,7 @@ export function getPackageManagerCommand(
         add: useBerry ? 'yarn add' : 'yarn add -W',
         addDev: useBerry ? 'yarn add -D' : 'yarn add -D -W',
         rm: 'yarn remove',
-        exec: useBerry ? 'yarn exec' : 'yarn',
+        exec: 'yarn',
```

Comment:
In `create-nx-workspace` this is set to `npx`... can you fix that one in a follow up PR as well please?